.gitignore

```diff
@@ -1,2 +1,3 @@
 node_modules
 .DS_Store
+dist
```

Comment:
You likely wouldn't have messed up the line ending here had you not deleted the `.editorconfig` file.

Reply:
Just trying to learn here... What do you mean the line ending is messed up? If the .gitignore file works, I could care less.